crypto/BRCryptoWalletManager.c

```diff
@@ -945,6 +965,11 @@ cryptoWalletManagerSubmitSigned (BRCryptoWalletManager cwm,
             // to wallet.  So, we'll add transfer here...
             cryptoWalletAddTransfer (wallet, transfer);
 
+            // Add the signed/submitted transfer to the GEN wallet.  If the submission fails
+            // we'll remove it then.  For now, include transfer when computing the balance and
+            // the sequence-number 
+            genWalletAddTransfer (cryptoWalletAsGEN(wallet), cryptoTransferAsGEN(transfer));
```

Comment:
OK - so it gets added once it is signed so we have the hash.  Get's removed if a failure happens.

Reply:
Yes